src/os/bluestore/NVMEDevice.cc

```diff
@@ -365,7 +365,12 @@ void SharedDriverQueueData::_aio_handle(Task *t, IOContext *ioc)
  again:
     dout(40) << __func__ << " polling" << dendl;
     if (current_queue_depth) {
-      spdk_nvme_qpair_process_completions(qpair, g_conf->bluestore_spdk_max_io_completion);
+      r = spdk_nvme_qpair_process_completions(qpair, g_conf->bluestore_spdk_max_io_completion);
```

Comment:
you need to call g_conf->get_val<uint64_t>("bluestore_spdk_max_io_completion") instead of legacy usage.

then remove change in legacy_config_opts.h,

Reply:
@yuyuyu101 Is this OK,  for other legacy opts, I will not update in this patch

Reply:
i'd suggest to use the new style option even for opts not introduced in this PR. we are switching from the old style to the new one anyway.